### PR TITLE
Jason Tan Lee

### PR DESCRIPTION
**Pull request by akorolczuk**

### CODE REVIEW

File: JasonTanLee/Assets/Scripts/Game/Environment/Trophy.cs [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+Project:    Tank Battalion
+Developer:  Neil Jason V. Tan Lee
+Company:    Bravo Company - jason.tanlee23@gmail.com
+Date:       01/10/2019 05:09
+===============================================================*/
+
+namespace TankBattalion.Game {
+
+    public class Trophy : MonoBehaviour, IDestroyable {
+
```
Comment (akorolczuk) [inline JasonTanLee/Assets/Scripts/Game/Environment/Trophy.cs]: This is really weird. why would Trophy responsibility be to load a scene?

File: JasonTanLee/Assets/Scripts/Tools/Variables/Generic/Vector3Variable.cs [hunk 2]
```diff
@@ -0,0 +1,44 @@
+using UnityEngine;
+
+/*===============================================================
+Project:    Tools
+Developer:  Neil Jason V. Tan Lee
+Company:    Jason Tan Lee - jason.tanlee23@gmail.com
+Date:       23/09/2019 16:28
+===============================================================*/
+
```
Comment (akorolczuk) [inline JasonTanLee/Assets/Scripts/Tools/Variables/Generic/Vector3Variable.cs]: This looks like our BindableProperties just with a lot of extra steps.. Could save a lot of time by using generics..

File: JasonTanLee/Assets/Scripts/Game/Environment/Trophy.cs (hunk 1 above)
Comment (owenmills) [reply JasonTanLee/Assets/Scripts/Game/Environment/Trophy.cs]: yeah, it's not great - but this is the end game condition right.  The Trophy has been hit, so load the main scene.

Comment (akorolczuk) [reply JasonTanLee/Assets/Scripts/Game/Environment/Trophy.cs]: still, trophy should notify some game manager or something. What if you decide to have multiple trophies?

**Pull request closed without merging**